src/app/signup/signup.component.html

```diff
@@ -0,0 +1,18 @@
+<app-authentication>
+  <form [formGroup]="authForm" (ngSubmit)="onSubmit(authForm.value)">
```

Comment:
Dont put value, obtain it on component with this.authForm.value
```suggestion
  <form [formGroup]="authForm" (ngSubmit)="onSubmit()">
```